packages/backend/src/scheduler.ts

```diff
@@ -67,4 +67,4 @@ createTerminus(server, {
     logger: Logger.error,
 });
 
-server.listen(process.env.PORT || 8081);
+server.listen(process.env.SCHEDULER_PORT || 8081);
```

Comment:
Can we make the changes without changing this env var? afaik, it's very common for deployment environments to set `$PORT` and expect it to be picked up by the application.